backend/src/main/java/com/ambacam/service/ActionService.java

```diff
@@ -0,0 +1,115 @@
+package com.ambacam.service;
+
+import com.ambacam.exception.ResourceBadRequestException;
+import com.ambacam.exception.ResourceNotFoundException;
+import com.ambacam.model.Action;
+import com.ambacam.repository.ActionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@Transactional(rollbackFor = Exception.class)
+public class ActionService {
+	@Autowired
+	private ActionRepository actionRepository;
+
+	/**
+	 * Create an action
+	 * 
+	 * @param action The new action to create
```

Comment:
Please write only @param action

Reply:
Please do the same for other functions

Reply:
That is a description of the parameter. I think it is a good practice to do that, since it is always helpful to get a description of a parameter.

http://www.oracle.com/technetwork/articles/java/index-137868.html

Reply:
In fact, it is also helpful a time to generate docs of APIs.

Reply:
Indeed!